rusoto/services/sts/src/custom/credential.rs

```diff
@@ -19,7 +19,7 @@ pub const DEFAULT_ROLE_DURATION_SECONDS: i32 = 900;
 
 /// Trait for conversions from STS Credentials to AWS Credentials.
 pub trait NewAwsCredsForStsCreds {
-    /// Creates an [AwsCredentials](../struct.AwsCredentials.html) from a [Credentials](struct.Credentials.html)
+    /// Creates an [AwsCredentials](../rusoto_credential/struct.AwsCredentials.html) from a [Credentials](struct.Credentials.html)
     /// Returns a [CredentialsError](../struct.CredentialsError.html) in case of an error.
```

Comment:
While we're in here, can you fix the link for `CredentialsError` here as well?

Reply:
Good catch. Added that fix to the PR.